encoding.bs

```diff
@@ -1660,20 +1631,18 @@ constructor steps are:
      <li><p>Let <var>outputChunk</var> be the result of running <a>serialize stream</a> with
      <var>decoder</var> and <var>output</var>.
 
-     <li><p>if <var>outputChunk</var> is non-empty, call
-     <a abstract-op>TransformStreamDefaultControllerEnqueue</a>(<var>controller</var>,
-     <var>outputChunk</var>).
+     <li><p>If <var>outputChunk</var> is non-empty, then <a for=TransformStream>enqueue</a>
+     <var>outputChunk</var> in <var>decoder</var>'s <a for=GenericTransformStream>transform</a>.
 
-     <li><p>Return a new promise resolved with undefined.
+     <li><p>Return.
     </ol>
 
    <li><p>Let <var>result</var> be the result of <a>processing</a> <var>token</var> for
    <var>decoder</var>'s <a for=TextDecoderCommon>decoder</a>, <var>decoder</var>'s
    <a for=TextDecoderCommon>stream</a>, <var>output</var>, and <var>decoder</var>'s
    <a for=TextDecoderCommon>error mode</a>.
 
-   <li><p>If <var>result</var> is <a>error</a>, then return a new promise rejected with a
-   {{TypeError}} exception.
+   <li><p>If <var>result</var> is <a>error</a>, then throw a {{TypeError}} exception.
```

Comment:
Should we drop the trailing " exception" here as we do in other standards?

Reply:
I don't have an opinion on this one.

Reply:
Let's drop it then, I can make this change as well before landing.